packages/editor/src/client/panel/teaserSelectPanel.tsx

```diff
@@ -21,13 +21,12 @@ export interface TeaserSelectPanelProps {
 export function TeaserSelectPanel({onClose, onSelect}: TeaserSelectPanelProps) {
   const [type, setType] = useState<TeaserType>(TeaserType.Article)
 
-  const [peerFilter, setPeerFilter] = useState<ArticleFilter>({title: ''})
+  // const [peerFilter, setPeerFilter] = useState<ArticleFilter>({title: ''})
```

Comment:
I know you haven't opened this up for review yet but I wanted to let you know that the peerFilter works a bit different and the filter in the peerArticles is not actually used (or even defined in the args).

peerFilter checks for the name of the peer:

```
.filter(peer => (peerFilter ? peer.name === peerFilter : true))
```